qase-python-commons/src/qase/commons/models/base.py

```diff
@@ -0,0 +1,8 @@
+import json
+
+from pathlib import PosixPath
+
+
+class Base:
```

Comment:
Maybe, call it BaseModel?

Will we add or move more shared code to this class?